rust/blockstore/src/arrow/block/delta/delta.rs

```diff
@@ -52,14 +52,6 @@ impl BlockDelta {
         V::delete(prefix, key.into(), self)
     }
 
-    /// Gets the minimum key in the block delta.
-    pub fn get_min_key(&self) -> Option<CompositeKey> {
```

Comment:
was this not being used anywhere?

Reply:
nope, but i added a flavor back in stacked pr